### ISSUE TITLE
Incompatibility with Tabbed Chatlog module

### ISSUE THREAD

**Issue by iconoplast** [RedReign/FoundryVTT-BetterRolls5e#249]
Minor issue but the tabbed chatlog module breaks when advantage is retroactively applied to a roll.  

Basically if you attack, then click the + button to roll an additional die, the chat card disappears.

**Comment by CarlosFdez** [comment]
Honestly, that sounds like a bug with the tabbed chatlog module. Does MARS have the same issue when you edit rolls?

**Comment by iconoplast** [comment]
Good point I'll let the TBC people know.  Haven't tried MARS.  Honestly I stopped using tabbed chat and don't miss it since we use voice chat but I can test with mars once I install it
